includes/admin/class.llms.admin.menus.php

```diff
@@ -5,7 +5,7 @@
  * @package LifterLMS/Admin/Classes
  *
  * @since 1.0.0
- * @version 5.3.1
+ * @version [version]
```

Comment:
Thank you for catching this. It looks like I had intended to autoload the tools, but since those class files instantiate themselves, I decided **not** to address that kind of issue in this round of autoloader changes.